Backend/routes/history.js

```diff
@@ -51,8 +51,21 @@ router.get('/:userId', async (req, res) => {
         res.json(history);
 
     }
-
 })
 
+router.get('/registered_days/:userId', async (req, res) => {
```

Comment:
Postman demostration of registered days and meal by date
https://www.loom.com/share/6819a927cd224e8db8d0bc5535d91fc7?sid=d8ef0356-1f2a-4dea-8b4e-d6d45e8efcf8